ssl/t1_lib.c

```diff
@@ -2780,16 +2775,23 @@ int ssl_parse_serverhello_tlsext(SSL *s, PACKET *pkt)
     return 1;
 }
 
+static RAW_EXTENSION *get_extension_by_type(RAW_EXTENSION *exts, size_t numexts,
+                                            unsigned int type)
+{
+    size_t loop;
+
+    for (loop = 0; loop < numexts; loop++) {
```

Comment:
Not sure I see the point here? There are only two uses of loop here anyway...it doesn't seem to simplify things at all? Similarly I'm not sure why counting down is better.